pkg/scheduler/framework/types.go

```diff
@@ -0,0 +1,820 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
```

Comment:
this file should have been `git mv`ed to preserve the history.

Reply:
also did `git mv` to all the other files in v1alpha1/

Reply:
this is still not `git mv`ed, we should be able to see the history if it was.

Reply:
@ahg-g I just saw this comment. Is this still an issue?

Reply:
I tried to verify with 

```
kubernetes$ git log --stat -M
commit f2777eed9a740382bbb90cc3e0440c3e6319eaf9 (HEAD -> split-scheduler, origin/split-scheduler)
Author: Ali <aliyfarah9@gmail.com>
Date:   Sat Oct 10 01:41:44 2020 +1100

    Move scheduler interface to pkg/scheduler/framework

 kind-config.yaml                                                                   |  28 +++
 pkg/scheduler/BUILD                                                                |   4 +-
 pkg/scheduler/core/BUILD                                                           |   4 +-
 pkg/scheduler/core/extender.go                                                     |   2 +-
 pkg/scheduler/core/extender_test.go                                                |   2 +-
 pkg/scheduler/core/generic_scheduler.go                                            |   2 +-
 pkg/scheduler/core/generic_scheduler_test.go                                       |  14 +-
 pkg/scheduler/factory.go                                                           |   2 +-
 pkg/scheduler/factory_test.go                                                      |   2 +-
 pkg/scheduler/framework/BUILD                                                      |  50 ++++
 pkg/scheduler/framework/{v1alpha1 => }/cycle_state.go                              |   2 +-
 pkg/scheduler/framework/{v1alpha1 => }/cycle_state_test.go                         |   2 +-
 pkg/scheduler/framework/{v1alpha1 => }/extender.go                                 |   2 +-
 pkg/scheduler/framework/{v1alpha1 => }/fake/BUILD                                  |   4 +-
 pkg/scheduler/framework/{v1alpha1 => }/fake/listers.go                             |   2 +-
 pkg/scheduler/framework/{v1alpha1 => }/interface.go                                |  10 +-
 pkg/scheduler/framework/{v1alpha1 => }/interface_test.go                           |   2 +-
 pkg/scheduler/framework/{v1alpha1 => }/listers.go                                  |   2 +-
[...]
 pkg/scheduler/framework/{v1alpha1 => }/types_test.go                               |   2 +-
[...]

```

I assumed the arrows ( =>) means it was moved.

Reply:
yes (=>) means moved, but I don't see the types.go in the list, it would also show up here in the diff if it was actually moved.

Reply:
ok i will look at it now.

Reply:
@ahg-g I don't think git allows me to mv `types.go`. I'm seeing what seems to be a Git issue:

Move framework/v1alpha1/types.go` to `framework/v1alpha1.`
Git will show moved correct. However as soon as I create the alias `types.go` file inside `framwork/v1alpha1`, git automatically converts the `framework/types.go` file from modified to created.

Reply:
ok, np.